jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ErrorPageErrorHandler.java

```diff
@@ -175,13 +175,16 @@ public String getErrorPage(HttpServletRequest request)
     /**
      *
      * @param t the initial exception
+     * @param matchedThrowable the class we found matching the error page (can be null)
      * @return the first non {@link ServletException} from root cause chain
      */
-    private Throwable getFirstNonServletException(Throwable t)
+    private Throwable getFirstNonServletException(Throwable t, Class<?> matchedThrowable)
```

Comment:
This name is not correct now.  How about:
```suggestion
    private Throwable unwrapServletException(Throwable t, Class<?> matchedThrowable)
```
```suggestion
    private Throwable getFirstNonServletException(Throwable t, Class<?> matchedThrowable)
```

Reply:
good idea!
done in 36438582c26a64e61a2c03325bb47f35b58b1a6d